nupic/research/frameworks/pytorch/imagenet/common_experiments.py

```diff
@@ -24,13 +24,16 @@
     ImagenetExperiment,
 )
 
-
 class RezeroedKWinnersImagenetExperiment(mixins.RezeroWeights,
                                          mixins.UpdateBoostStrength,
                                          ImagenetExperiment):
     pass
 
+class KnowledgeDistillationImagenetExperiment(mixins.KnowledgeDistillation,
```

Comment:
Is this experiment using sparse weights and k-winners?
If it is then you need to add `mixins.RezeroWeights` and  `mixins.UpdateBoostStrength` to the class